app/Repositories/AdminRepository.php

```diff
@@ -18,22 +18,16 @@ class AdminRepository
 {
 
 
-
     /**
      * @param Request $request
-     * @return Response
+     * @return
      */
     public function createAdmin(Request $request)
```

Comment:
This should be more of a UserRepository since both admins and staff users are created here. Please also update the names to createUser and buildUserProperties from "buildAdminProperties"